api/tests/unit/tooling/database-builder/database-builder_test.js

```diff
@@ -312,7 +336,7 @@ describe('Unit | Tooling | DatabaseBuilder | database-builder', () => {
       ];
 
       // when
-      await databaseBuilder.commit();
+      await catchErr(databaseBuilder.commit, databaseBuilder)();
 
```

Comment:
Question: Du coup, si on ne check pas l'erreur dans ce test, a-t-on besoin de catchErr ?

Reply:
On est dans un cas où ça va échouer, on peut le remplacer par :
```
  try {
    await databaseBuilder.commit();
  }
  catch() {
  }
```
Je trouve la version `catchErr()` plus concise, et en vrai les deux me vont.

Ce que je me dis aussi c'est que ce test en particulier vérifie un changement d'état interne, c'est un peu fragile et peu expressif. On pourrait vérifier le comportement que ce changement d'état permet, ce serait plus expressif et plus robuste au refactoring.

Reply:
Rien à voir mais je trouve que l'écriture
```js
catchErr(() => databaseBuilder.commit())
```
est plus claire que
```js
catchErr(databaseBuilder.commit, databaseBuilder)
```
avec son horrible répétition qui nous rappelle que `this` est si particulier en JS… 😢 
Pas vous ?

Reply:
…et peut-être qu'[un jour](https://github.com/tc39/proposal-bind-operator) on écrira
```js
catchErr(::databaseBuilder.commit)
```
😉